chsdi/views/search.py

```diff
@@ -98,17 +98,18 @@ def _fuzzy_search(self, searchTextFinal):
 
     def _swiss_search(self):
         limit = self.limit if self.limit and self.limit <= self.LOCATION_LIMIT else self.LOCATION_LIMIT
-        self.sphinx.SetLimits(0, limit)
-
         # Define ranking mode
         if self.bbox is not None and self.sortbbox:
             coords = self._get_geoanchor_from_bbox()
             self.sphinx.SetGeoAnchor('lat', 'lon', coords[1], coords[0])
             self.sphinx.SetSortMode(sphinxapi.SPH_SORT_EXTENDED, '@geodist ASC')
+            limit = 150
```

Comment:
I would set this variable at the class level, like the other one

Reply:
done